src/server/handlers/schema.js

```diff
@@ -18,6 +18,6 @@ export const repo = new schema.Entity('repos', {
 export const repoList = new schema.Array(repo);
 
 export const snap = new schema.Entity('snaps', {}, {
-  idAttribute: 'name'
+  idAttribute: 'git_repository_url'
```

Comment:
@bartaz: This was a dilemma I faced creating this PR. The repository URL is already used as a unique identifier by the Launchpad API endpoint, so it makes sense to maintain this consistency in the frontend. For now.

I think `name` would make a suitable identifier for snaps, but it affects a much greater part of the application than the Redux store and should be out-of-scope for this task.